AGENTS.md

```
@@ -0,0 +1,99 @@
+```bash
+cd verify-ui
+npm install
+npm start          # generates env.config.js from .env, then runs react-app-rewired start
+                   # NOTE: prestart runs all tests first; use react-app-rewired start directly to skip
+npm test           # jest with coverage and snapshot updates
+npm run test:ci    # jest in CI mode (used by prestart/prebuild)
+npm run build      # runs tests then react-app-rewired build
+```
+
+To skip the prestart test run during development:
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)